### PR TITLE
Troubleshooting ServiceControl InvalidOperationException: Sequence contains no matching element

### PR DESCRIPTION
**Pull request by ramonsmits**

### CODE REVIEW

File: servicecontrol/sequence-contains-no-matching-element.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+title: ServiceControl - InvalidOperationException: Sequence contains no matching element
+summary: ServiceControl Management Utility shows System.InvalidOperationException: Sequence contains no matching element
+reviewed: 2024-06-06
+---
+
+## Symptoms
+
+The following error is shown when launching the ServiceControl Management Utility (SCMU):
```
Comment (jpalac) [inline servicecontrol/sequence-contains-no-matching-element.md:9]: Would be worth mentioning that this error was also occuring during the instance update process when upgrading 5.x to anything.

Comment (jpalac) [inline servicecontrol/sequence-contains-no-matching-element.md:9]:
```suggestion
The following error is shown when either:
 - launching the ServiceControl Management Utility (SCMU)
 - upgrading an instance on version >= 5.x via the SCMU

```

**Review by jpalac — APPROVED** [review]

File: servicecontrol/sequence-contains-no-matching-element.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+- Uninstall the version 4.x via Windows Add/Remove software.
+
+
+If you still have version 4.x instances:
+
+- Install the latest version 4 update which does not have this bug
+
+
+The lastest version 4.x release at this writing is 4.33.3 which is available for download at the following location:
```
Comment (abparticular) [inline servicecontrol/sequence-contains-no-matching-element.md:41]:
```suggestion
The lastest version 4.x release at this writing is 4.33.4 which is available for download at the following location:

```

File: servicecontrol/sequence-contains-no-matching-element.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+- Uninstall the version 4.x via Windows Add/Remove software.
+
+
+If you still have version 4.x instances:
+
+- Install the latest version 4 update which does not have this bug
+
+
+The lastest version 4.x release at this writing is 4.33.4 which is available for download at the following location:
```
Comment (johnsimons) [inline servicecontrol/sequence-contains-no-matching-element.md:41]:
```suggestion
The latest version 4.x release at this writing is 4.33.4, which is available for download at the following location:

```

**Pull request merged**